src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1276,7 +1276,10 @@ public function getMethod()
                 if ($method = $this->headers->get('X-HTTP-METHOD-OVERRIDE')) {
                     $this->method = strtoupper($method);
                 } elseif (self::$httpMethodParameterOverride) {
-                    $this->method = strtoupper($this->request->get('_method', $this->query->get('_method', 'POST')));
+                    $method = $this->request->get('_method', $this->query->get('_method', 'POST'));
+                    if (\is_string($method)) {
```

Comment:
what should happen when a non-string is found here?
returning it as is looks strange, isn't it? it break the method's signature somehow.

Reply:
Oh, indeed :)